eth-contracts/contracts/CKBChainV2.sol

```diff
@@ -69,6 +70,17 @@ contract CKBChainV2 is ICKBChain, ICKBSpv {
     /// Known headers. Stores up to `finalized_gc_threshold`.
     mapping(bytes32 => BlockHeader) blockHeaders;
 
+    // refer to https://github.com/ethereum/EIPs/blob/master/EIPS/eip-712.md
+    uint public constant SIGNATURE_SIZE = 65;
+    uint public constant VALIDATORS_SIZE_LIMIT = 20;
+    string public constant name = "Force Bridge CKBChain";
+    // ADD_HEADERS_TYPEHASH = keccak256("AddHeaders(bytes data)");
```

Comment:
The signature of the `add_header` function is `AddHeaders(bytes data, bytes signatures)`, but the hash we used here is `AddHeaders(bytes data)`

Reply:
In eip-712, when calc TYPEHASH, the signature of the function should not consider the `signatures` or `r, s, v` paramters,  I think this is also the norm.